images/gcsfuse/Dockerfile

```
@@ -0,0 +1,8 @@
+FROM ubuntu:22.04
+
+ARG VERSION=${VERSION:-1.0}
+
+RUN GCSFUSE_REPO=gcsfuse-$(lsb_release -c -s) && \
+    echo "deb http://packages.cloud.google.com/apt $GCSFUSE_REPO main" | tee /etc/apt/sources.list.d/gcsfuse.list && \
```

This needs to be https; see [install docs](https://cloud.google.com/storage/docs/gcsfuse-install#install-binaries) and GoogleCloudPlatform/gcsfuse#1424.

```suggestion
    echo "deb https://packages.cloud.google.com/apt $GCSFUSE_REPO main" | tee /etc/apt/sources.list.d/gcsfuse.list && \
```